index.html

```diff
@@ -1,76 +1,85 @@
 <!DOCTYPE html>
-<html lang="kr">
+<html lang="ko">
    <head>
       <meta charset="UTF-8">
       <meta name="viewport" content="width=device-width, initial-scale=1.0">
+      <title>Lnikbrary</title>
+
+      <!-- 페이스북, 카카오톡, 인스타 공유-->
       <meta property="og:title" content="Linkbrary">
       <meta property="og:image" content="Assets/PreviewImage.png">
       <meta property="og:url" content="Rhajiit.git">
       <meta property="og:description" content="세상의 모든 정보를 쉽게 저장하고 관리해 보세요">
-      <link rel="stylesheet" href="Style.css">
+      <!-- 트위터 공유-->
+      <meta name="twitter:title" content="Linkbrary">  
+      <meta name="twitter:image" content="Assets/PreviewImage.png">
+      <meta name="twitter:url" content="Rhajiit.git">
+      <meta name="twitter:description" content="세상의 모든 정보를 쉽게 저장하고 관리해 보세요">
+
+      <link rel="stylesheet" href="Styles/reset.css">
+      <link rel="stylesheet" href="Styles/defaultStyle.css">
+      <link rel="stylesheet" href="Styles/landingPage.css">
       <link rel="preconnect" href="https://fonts.googleapis.com">
       <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
       <link href="https://fonts.googleapis.com/css2?family=Acme&family=Noto+Sans+KR:wght@400;700&family=Poppins:wght@400;600&display=swap" rel="stylesheet">
    </head>
    <body>
-      <nav class="landingPage">
-         <div class="landingPage">
-            <a href="index.html"><img src="Assets/logo.svg" alt="Linkbrary"></a>
-            <a href="signIn.html" class="button">로그인</a>
+
+      <header>
+         <div class="flexibleHeaderInTablet">
+            <a href="index.html"><img src="Assets/logos/logo.svg" alt="Linkbrary"></a>
+            <a href="signIn.html" class="buttonBasicStyle">로그인</a>
```

Comment:
classname에 style은 안 넣어도 될 것 같아요. 넣었을 때 이점이 있는 지 잘 모르겠어서요.
```suggestion
            <a href="signIn.html" class="buttonBasic">로그인</a>
```